app/Relations/HasAlbumThumb.php

```diff
@@ -16,6 +16,9 @@
 use Illuminate\Database\Query\Builder as BaseBuilder;
 use Illuminate\Database\Query\JoinClause;
 
+/**
+ * @method where($column, $operator, $value)
+ */
```

Comment:
Found the problem, but instead of adding a single method I solved the problem more generally in https://github.com/LycheeOrg/Lychee/pull/1336/commits/17979a0823a687464869b49205a2c33b12429fa5.